wolftpm/tpm2_wrap.h

```diff
@@ -2272,6 +2272,70 @@ WOLFTPM_API int wolfTPM2_CreateAndLoadAIK(WOLFTPM2_DEV* dev, WOLFTPM2_KEY* aikKe
 */
 WOLFTPM_API int wolfTPM2_GetTime(WOLFTPM2_KEY* aikKey, GetTime_Out* getTimeOut);
 
+#if !defined(WOLFTPM2_NO_WOLFCRYPT) && defined(WOLFSSL_CERT_GEN)
+/*!
+    \ingroup wolfTPM2_Wrappers
+    \brief Helper for Certificate Signing Request (CSR) generation using a
+        wolfCrypt key (RsaKey or ecc_key)
```

Comment:
This seems strange.  Why would you need a wolfcrypt key?  Isn't the key in the TPM? Below in param description it says its a loaded WOLFTPM2_KEY structure.